modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/partials/multiple_selection.jsp

```diff
@@ -0,0 +1,44 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<form action="<%= fdsSampleDisplayContext.getPortletURL() %>" method="post" name="fm">
+	<input name="<%= Constants.CMD %>" type="hidden" value="<%= Constants.DELETE %>" />
+	<input name="redirect" type="hidden" value="<%= String.valueOf(fdsSampleDisplayContext.getPortletURL()) %>" />
+
+	<frontend-data-set:headless-display
+		additionalProps='<%=
+			HashMapBuilder.<String, Object>put(
+				"greeting", "Hello"
+			).build()
+		%>'
+		apiURL="<%= fdsSampleDisplayContext.getAPIURL() %>"
+		bulkActionDropdownItems="<%= fdsSampleDisplayContext.getBulkActionDropdownItems() %>"
+		customViewsEnabled="<%= true %>"
+		fdsActionDropdownItems="<%= fdsSampleDisplayContext.getFDSActionDropdownItems() %>"
+		formId="fm"
+		id="<%= FDSSampleFDSNames.FDS_SAMPLES %>"
+		itemsPerPage="<%= 20 %>"
+		namespace="<%= liferayPortletResponse.getNamespace() %>"
+		pageNumber="<%= 1 %>"
+		portletURL="<%= liferayPortletResponse.createRenderURL() %>"
+		propsTransformer="js/SampleFDSPropsTransformer"
+		selectedItemsKey="id"
+		selectionType="multiple"
+		style="fluid"
+	/>
```

Comment:
If we have a tab that showcases only multiple selection, we should remove all the noise attributes. It should be minimal configuration + anything required for multiple selection. I believe that's only `selectionType`.